.github/workflows/checks.yaml

```
@@ -0,0 +1,38 @@
+name: Checks
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+    branches: [master]
+jobs:
+  count_commits:
+    name: Check that there is only one commit
```

Suggest something snappier:
```suggestion
    name: Enforce single commit PR
```

---

Name changed.